test/functional/tools/for_workflows/create_input_collection.xml

```diff
@@ -0,0 +1,42 @@
+<tool id="create_input_collection" name="create_input_collection" version="0.1.0">
+  <description>This tool is used to create a collection of text files.</description>
+  <command detect_errors="exit_code">
+    mkdir outputs; cd outputs; python $script
+  </command>
+  <configfiles>
+    <configfile name="script">
+import os
+
+for i in range($collection_size):
+    template = "File number %s\n"
+    contents = template % i
+    with open(str(i), "w") as f:
+        f.write(contents)
+</configfile>
```

Comment:
Then I'd do something like https://bitbucket.org/crs4/orione-tools/src/d39acf781fdd5e8ce78382d19520e5c225b81086/custom/exomedepth/exomedepth.xml?at=bb-orione-tools&fileviewer=file-view-default#exomedepth.xml-32 but this is not really that important.